packages/bot-web-ui/src/components/__tests__/bot-stopped.spec.tsx

```diff
@@ -82,4 +82,26 @@ describe('<BotStopped />', () => {
             expect(global.location.reload).toHaveBeenCalled();
         });
     });
+
+    it('should render onClickClose on press of enter keydown', async () => {
+        const mockEventListener = jest.fn();
+        document.addEventListener('keydown', mockEventListener);
+        render(<BotStopped />, {
+            wrapper,
+        });
+        const close_button = screen.getByTestId('data-close-button');
+        userEvent.type(close_button, '{Enter}');
+        expect(mockEventListener).toHaveBeenCalledWith(expect.objectContaining({ key: 'Enter', code: 'Enter' }));
+    });
+
+    it('should render onClickClose on press of enter keydown', async () => {
```

Comment:
in all tests here we don't need `async`
```suggestion
    it('should render onClickClose on press of enter keydown', () => {
```